src/Sparrow/Json/JsonOperationContext.cs

```diff
@@ -160,6 +165,10 @@ void TryExecute(Action a)
             if (_perCoreLazyStringValuesList.TryPull(out _allocateStringValues) == false)
                 _allocateStringValues = new FastList<LazyStringValue>(256);
 
+            MetadataIdFieldCache = GetLazyStringForFieldWithCaching("@id");
+            MetadataKeyFieldCache = GetLazyStringForFieldWithCaching("@metadata");
+            MetadataCollectionFieldCache = GetLazyStringForFieldWithCaching("@collection");
```

Comment:
Are we always going to want to do that? Wouldn't it make more sense to do this lazily?

Reply:
I do not like this change, especially because we are clearly mixing layers here (pushing to Sparrow strings from upper layers - clearly visible, because Constants cannot be used). We need to find a better way to do this IMO.